OscLib/Makefile

```diff
@@ -2,19 +2,19 @@
 
 # Record what flags were used. Can dump with
 # readelf -p .GCC.command.line libOscLib.so
-CFLAGS := -std=c++17 -I.. -I${ROOT_INC} -I${EIGEN_INC} -I${BOOST_INC} -I${GSL_INC} -g -Wall -Wpedantic -Wextra -Werror -grecord-gcc-switches
+CFLAGS := -std=c++17 -I.. -I${ROOT_INC} -I${EIGEN_INC} -g -Wall -Wpedantic -Wextra -Werror -grecord-gcc-switches
```

Comment:
why are boost and gsl includes dropped here? Both have users in the code. Boost tends to work by accident because there's usually a system boost library, but that's not really what you want.

Reply:
for some reason I thought these were only used by Stan, so I moved them down into the `#ifdef STAN` block later.  But if that's not true, this part can certainly be reverted.

Reply:
No, I grepped for them and they are in PMNS_CPT or something.